addons/account/wizard/account_payment_register.py

```diff
@@ -380,14 +380,10 @@ def _compute_currency_id(self):
     @api.depends('payment_type', 'company_id', 'can_edit_wizard')
     def _compute_available_journal_ids(self):
         for wizard in self:
-            if wizard.can_edit_wizard:
-                batch = wizard._get_batches()[0]
-                wizard.available_journal_ids = wizard._get_batch_available_journals(batch)
-            else:
-                wizard.available_journal_ids = self.env['account.journal'].search([
-                    ('company_id', '=', wizard.company_id.id),
-                    ('type', 'in', ('bank', 'cash')),
-                ])
+            available_journal_ids = self.env['account.journal']
+            for batch in wizard._get_batches():
+                available_journal_ids |= wizard._get_batch_available_journals(batch)
+            wizard.available_journal_ids = wizard._get_batch_available_journals(batch)
```

Comment:
```suggestion
            available_journals = self.env['account.journal']
            for batch in wizard._get_batches():
                available_journals |= wizard._get_batch_available_journals(batch)
            wizard.available_journal_ids = [Command.set(available_journals.ids)]
```

Reply:
oh my...it seems I forgot to change the last line :sweat_smile: